packages/reference/environments/production/us-east-2/mongodb_atlas_identity_provider/terragrunt.hcl

```diff
@@ -0,0 +1,24 @@
+include "panfactum" {
+  path   = find_in_parent_folders("panfactum.hcl")
+  expose = true
+}
+
+terraform {
+  source = include.panfactum.locals.pf_stack_source
+}
+
+dependency "authentik_atlas_mongodb_sso" {
+  config_path = "../authentik_mongodb_atlas_sso"
+}
+
+inputs = {
+  federation_settings_id = "679ac90e37161e292db2780e"
+  idp_id = "67a80b37499ddf3676fa8f2b"
+  organization_id = "679ac9030691076f53402259"
+
+  issuer_url = dependency.authentik_atlas_mongodb_sso.outputs.issuer_url
+  sso_url = dependency.authentik_atlas_mongodb_sso.outputs.url_sso_post
+
```

Comment:
Shouldn't this have `sso_debug_enabled` to `false` in order to follow the guide's recommendations. Make a note that it should be set to `true` on first install.